EventCatalogAPI/Controllers/EventItemsController.cs

```diff
@@ -208,9 +209,46 @@ private List<EventItem> ChangeImageUrl(List<EventItem> items)
             return items;
         }
 
+        [HttpGet("[action]/Catagory/{catagoryId}/Type{typeId}")]
```

Comment:
I like the idea of this addition - have you tested it yet? Let's test before we merge.

Reply:
I tested it  and it works for me  but with some adjustment   ,  I push the update   made  based on the comment